workers/data_refinery_workers/processors/smasher.py

```diff
@@ -321,7 +391,8 @@ def _upload(job_context: Dict) -> Dict:
         logger.exception("Failed to upload smash result file.", file=job_context["output_file"])
         job_context['job'].success = False
         job_context['job'].failure_reason = str(e)
-        job_context['success'] = False
+        # Delay failing this pipeline until the failure notify has been sent
+        # job_context['success'] = False
```

Comment:
Is this line meant as a way of conveying that it wasn't included on purpose?

Reply:
Yes